extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -217,15 +218,28 @@ export class InstrumentationService {
       return;
     }
 
-    // TODO(avimehta, #1297): Add all the listeners so that visibility
-    // conditions are monitored and callback is called when the conditions
-    // are met.
+    if (!config['visibilitySpec']) {
```

Comment:
Mind flipping the conditionals? `!else` double-negatives make everything confusing.